api/src/services/lottery.service.ts

```diff
@@ -956,6 +974,68 @@ export class LotteryService {
     return formattedActivityLogs;
   }
 
+  /**
+    runs the job to auto expire lotteries that are passed their due date
+    will call the the cache purge to purge all listings as long as updates had to be made
+  */
+  async autoPublishResults(): Promise<SuccessDTO> {
+    this.logger.warn('autoPublishLotteryResults job running');
+    await this.listingService.markCronJobAsStarted(
+      LOTTERY_PUBLISH_CRON_JOB_NAME,
+    );
+    const tomorrow = dayjs(
+      `${new Date().toISOString().split('T')[0]}T00:00:00.000Z`,
+    )
+      .add(1, 'days')
+      .toDate();
```

Comment:
This works for setting timezone. dayjs also supports timezone functionality as well https://day.js.org/docs/en/timezone/timezone and more specifically the utc plugin https://day.js.org/docs/en/parse/utc.

Reply:
I was more concerned that it correctly set the time to 00:00:00 so including this timezone this way avoids adding another function call